From faad12d22f497f31ab58ee29b237ccda991d9b59 Mon Sep 17 00:00:00 2001
From: Wu Jingchen <oven.wu@rock-chips.com>
Date: Thu, 23 Apr 2020 17:00:21 +0800
Subject: [PATCH] ARM: rockchip_defconfig: Add SONY drivers to fix
 CtsHardwareTestCases

android.hardware.input.cts.tests.SonyDualshock4TestCase#testAllKeys
android.hardware.input.cts.tests.SonyDualshock4TestCase#testAllMotions

Change-Id: I87beb10acf3afc42029fd5cf75907934707bdbb6
Signed-off-by: Wu Jingchen <oven.wu@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 34e39f4b0881..00f40b47569e 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -538,6 +538,7 @@ CONFIG_SND_SOC_TC358749X=y
 CONFIG_SND_SIMPLE_CARD=y
 CONFIG_HIDRAW=y
 CONFIG_UHID=y
+CONFIG_HID_SONY=y
 CONFIG_USB_HIDDEV=y
 CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
 CONFIG_USB_MON=y
@@ -570,6 +571,8 @@ CONFIG_MMC=y
 CONFIG_MMC_BLOCK_MINORS=32
 CONFIG_MMC_DW=y
 CONFIG_MMC_DW_ROCKCHIP=y
+CONFIG_NEW_LEDS=y
+CONFIG_LEDS_CLASS=y
 CONFIG_LEDS_GPIO=y
 CONFIG_LEDS_TRIGGER_BACKLIGHT=y
 CONFIG_LEDS_TRIGGER_DEFAULT_ON=y
-- 
2.35.3

